doc/chapters/filesystem/index.rst

```diff
@@ -0,0 +1,102 @@
+Filesystems
+***********
+
+This chapter documents the behavior and layout of the basic filesystem of the
+containers and where it picks up the different packages from. 
+
+
+Standard filesystem
+===================
+
+The base filesystem is bind mounted from a directory created by lxc, the
+directory is empty by default and is setup by the ``lxc-softwarecontainer.in``
+script, which is run by lxc init. 
+
+* The script creates several basic directories creating a normal unix
+  environment as defined by Filesystem Hierarchy Standard (FHS). 
+* The filesystem is still empty except for directories. 
```

Comment:
This is incorrect, since passwd, resolv.conf etc are present. Perhaps **mostly** empty would be a better wording.